### PR TITLE
Rename files, file paths

### PR DESCRIPTION
**Pull request by joeylouise**
Add public/ to file paths for index.html and example.js
Remove project from end of lesson2.md
Relates #6 #7 #8

### CODE REVIEW

File: LESSON2.md [hunk 1]
```diff
@@ -361,8 +361,3 @@ stdin.on('end', () => {
 
 *This is exactly the same as our read stream example at the top of this readme, as
 process.openStdin() is a read stream!*
-
-### Project
-
-With what we have covered in these two lessons, we now have everything we need to build some command-line tools! Move onto
```
Comment (bradreeder) [inline LESSON2.md]: @joeylouise I'm happy with all the other changes (thanks! ✨ ) but am unsure if we need to remove the project.md. I'd prefer to keep it in for the benefit of those who use this workshop outside of a FAC classroom. 

Could you tell the students to finish where they are if they get to the end of lesson 2 as the project will be covered in the research afternoon? If you wanted to include the project in the master-reference you could provide a link to the project file here from the research-afternoon.md file.

**Review by bradreeder — CHANGES_REQUESTED** [review]

File: LESSON2.md (hunk 1 above)
Comment (jsms90) [reply LESSON2.md]: How about including a line like "if you're completing the course at Founders and Coders, you will cover this later"?

**Comment by joeylouise** [comment]
@bradreeder 👍

File: LESSON2.md (hunk 1 above)
Comment (bradreeder) [reply LESSON2.md]: @jsms90 @joeylouise Sorry to be a pain, I'm just unsure if this is the best approach for updating resources in the long-term. If next cohort the mentors decide to not cover the project in the research afternoon, or use only one lesson of the workshop as Marko did, then we'd need to remember to update this line to reflect that.

I think whoever delivers a workshop should tell the students what parts of it they want them to cover. Happy to go with this if you both disagree though, and thanks for making the changes Joey. ( :

Comment (joeylouise) [reply LESSON2.md]: @bradreeder that makes sense, I'm happy to leave it as it was previously - I guess it just needs to be clear to those delivering this (i.e. in the future, all over the world..) what their options are

**Comment by joeylouise** [comment]
Hey @bradreeder I've restored the project bit back to the way it was, so now this PR is just renaming files and file paths

**Review by bradreeder — APPROVED** [review]

**Comment by bradreeder** [comment]
Thanks @joeylouise ✨

**Pull request merged**